substrate/runtime/consensus/src/lib.rs

```diff
@@ -96,6 +100,14 @@ decl_module! {
 	}
 }
 
+decl_storage! {
+	trait Store for Module<T: Trait> as Consensus {
+		// Authorities set actual at the block execution start. IsSome only if
+		// the set has been changed.
+		SavedAuthorities get(saved_authorities): default Vec<T::SessionKey>;
```

Comment:
Consider rename to `OriginalAuthorities`

Reply:
Also consider removing `default`, to ensure it's `Some` in the case that the set changed and `None` otherwise.